src/nspawn/nspawn-mount.c

```diff
@@ -600,6 +610,47 @@ int mount_custom(
         return 0;
 }
 
+// Retrieve existing subsystems. This function is called in a new cgroup namespace.
+static int get_controllers(Set *subsystems)
+{
```

Comment:
We pretty much have a function line this already in cg_kernel_controllers(), no?